Layer descriptions

ZachTRice opened the pull request:
Layer descriptions contained within am "i" button in the sidebar. If no description is available a disabled "i" button is shown. If a description is available, clicking the "i" button expands a modal box. The "i" button and modal box are only available within the desktop view.
Connects to #65
connects to #316

Benjaki2 commented on web/index.html:
```diff
@@ -382,13 +383,13 @@
         </div>
         <div id="selected-category">
-          
+
```
I'd leave out these spaces if there isn't anything inside the divs `<div></div>`

ZachTRice replied:
I agree. This is how it is currently on master, though I should have cleaned it up while I was working in this area. I'm not sure why git is picking this up as a change.

ZachTRice replied:
Ah, the differences is that there were spaces after the line return that are no longer present.

Benjaki2 replied:
ahhh makes sense. Thanks for cleaning up those spaces.

Benjaki2 commented on web/js/layers/wv.layers.info.js:
```diff
@@ -0,0 +1,85 @@
+        renderDescription($dialog);
+
+
+        var names = models.layers.getTitles(layer.id);
```
declare vars at top of scope

```
var loaded = function(custom) {
     var names;
```

Benjaki2 commented on web/js/layers/wv.layers.info.js:
```diff
@@ -0,0 +1,85 @@
+            .attr("id", "wv-layers-info-dialog")
+            .attr("data-layer", layer.id);
+        renderDescription($dialog);
+
```
excess line

ZachTRice commented:
Thanks, I appreciate the nit picky! 

Couple things I added to that last commit. Removed a `</div>` after the opening `<body>` tag that's been in master. Also, just noticed I had to tweak the css a bit to add back in the `li` / `a` elements to the class selectors since bank.css is setting these and my hover state colors were being over written with the default link color. 

At some point, I'd also like to remove as much element.class selector's as possible. Just to make scoping and compartmentalizing the css a bit easier.

Benjaki2 commented:
Wait till after next release to merge

ZachTRice commented:
Closing this PR because this feature will be merged into a new branch with search-descriptions. The new combined feature branch will then have a PR into development.
